openedx/core/djangoapps/notifications/utils.py

```diff
@@ -35,18 +32,7 @@ def get_show_notifications_tray(user):
     """
     Returns show_notifications_tray as boolean for the courses in which user is enrolled
     """
-    show_notifications_tray = False
-    learner_enrollments_course_ids = CourseEnrollment.objects.filter(
-        user=user,
-        is_active=True
-    ).values_list('course_id', flat=True)
-
-    for course_id in learner_enrollments_course_ids:
-        if SHOW_NOTIFICATIONS_TRAY.is_enabled(course_id):
-            show_notifications_tray = True
-            break
-
-    return show_notifications_tray
+    return True
```

Comment:
This method serves no purpose, please consider removing it

Reply:
I think we should check if `ENABLE_NOTIFICATIONS` is enabled in this method.